.travis.yml

```diff
@@ -7,25 +7,19 @@ matrix:
     - go: 1.11.x
       env: GO111MODULE=off
     - go: 1.11.x
-      env:
-      - GO111MODULE=on
-      - VET=1
+      env: GO111MODULE=on
     - go: 1.12.x
       env: GO111MODULE=off
     - go: 1.12.x
       env: GO111MODULE=on
     - go: 1.13.x
       env:
       - GO111MODULE=on
-      - GOPROXY=direct
+      - VET=1
     - go: 1.14.x
-      env:
-      - GO111MODULE=on
-      - GOPROXY=direct
```

Comment:
What's different that we now want to use the default proxy here? It's been around since 1.13 came out, right?

Reply:
Sorry for lack of context. Some of this cleanup was just making things consistent with other repos. I've removed the `GOPROXY=direct` so that CI will use the Go proxy servers, instead of always trying to directly source control.

This env var was added once upon a time when CI was red on "tip" version (maybe some instability?). Now that things seem stable on all recent versions (basically 1.13 and up), I figured using the default Go proxy was reasonable.